bpf/bpf_features_legacy.h

```diff
@@ -0,0 +1,10 @@
+#ifndef BPF_FEATURES_LEGACY_H_
+#define BPF_FEATURES_LEGACY_H_
+
+#define HAVE_SKB_CHANGE_TAIL
```

Comment:
Should be able to switch this one to the new format too.

Reply:
Good point

Reply:
To be honest, I don't even see skb_change_tail used anywhere and the check for it is commented out:

https://github.com/cilium/cilium/blob/master/bpf/run_probes.sh#L130

So I will just remove it.

Reply:
I think it's commented out because it became part of the regular `probe_run_ll()` case:

```
# grep CHANGE_TAIL /var/run/cilium/state/globals/bpf_features.h 
#define HAVE_SKB_CHANGE_TAIL
# git grep change_tail
bpf/include/bpf/api.h:static int BPF_FUNC(skb_change_tail, struct __sk_buff *skb, uint32_t nlen,
bpf/include/linux/bpf.h: * int bpf_skb_change_tail(struct sk_buff *skb, u32 len, u64 flags)
bpf/include/linux/bpf.h:        FN(skb_change_tail),            \
bpf/lib/icmp6.h:                if (skb_change_tail(skb, skb->len + trimlen, 0) < 0)
bpf/lib/icmp6.h:                if (skb_change_tail(skb, skb->len + trimlen, 0) < 0)
bpf/probes/raw_change_tail.t: * 5293efe62df8 ("bpf: add bpf_skb_change_tail helper")
bpf/probes/raw_change_tail.t:                   BPF_EMIT_CALL(BPF_FUNC_skb_change_tail),
bpf/probes/raw_change_tail.t:           .warn = "Kernel does not support bpf_skb_change_tail() helper. "
bpf/run_probes.sh:#probe_run_tc "skb_change_tail.c" "HAVE_SKB_CHANGE_TAIL"
```